latex2sympy.py

```diff
@@ -68,6 +72,23 @@ def process_sympy(sympy, variable_values={}):
     return return_data
 
 
+def pre_process_latex(latex):
+    '''
+    pre-processing for issues the parser cannot handle
+    '''
+
+    # find any single char sup/sub and wrap them in "{}"
+    # e.g. `4^26^2` => `4^{2}6^{2}`
+    # e.g. `x_22` => `x_{2}2`
+
+    # NOTE: this does modify variable names, but see `atom.VARIABLE()` processing below where this is reverted.
+    # this appears to be the easiest way without making the regex here too complex
+
+    pre_processed_latex = re.sub(r'([\^_])([0-9a-zA-Z])', '\\1{\\2}', latex)
```

Comment:
Not super familiar with this codebase. Is there any chance the latex would have  characters outside of `0-9a-zA-Z` that we'd need to account for here at all?

Reply:
@jkdufair I don't think so. This is specifically for when the sub/sup is a single character, so any latex commands would already be wrapped as they are always more than one char.